L1Trigger/TrackFindingTracklet/interface/Settings.h

```diff
@@ -968,6 +981,44 @@ namespace trklet {
 
     double stripLength_PS_{0.1467};
     double stripLength_2S_{5.0250};
+
+    //Variable bin edges for 1 to 12 bins.
```

Comment:
This adds a lot of complexity and numbers. Can we just give the binning for your preferred number of bins? (And perhaps just the bins on the +ve side, since symmetric).

Reply:
I reduced the binning to 6 bins, but I wasn't sure how to institute only keeping the positive side with respect to all the functions that used it.